transform/macros/map_maf_tiger_feature_class.sql

```diff
@@ -0,0 +1,16 @@
+{% macro map_maf_tiger_feature_class(maf_tiger_feature_class, k, v) -%}
+
+{% set maf_tiger_feature_class_dict = {
+    "G4110" : "Incorporated Place",
+    "G4210" : "Census Designated Place",
+    "G5040" : "Tabulation Block"
```

Comment:
Are there any other options? Is it worth doing an `ELSE` clause?

Reply:
Nope and nope!